src/pages-and-resources/discussions/app-list/FeaturesTable.test.jsx

```diff
@@ -1,13 +1,88 @@
+import React from 'react';
+import { IntlProvider } from '@edx/frontend-platform/i18n';
+import { render, queryAllByText, queryAllByTestId } from '@testing-library/react';
+
+import FeaturesTable from './FeaturesTable';
+import messages from './messages';
+
 describe('FeaturesTable', () => {
+  let apps;
+  let features;
+  let container;
+
+  beforeEach(() => {
+    apps = [
+      {
+        documentationUrl: 'http://example.com',
+        featureIds: ['discussion-page', 'embedded-course-sections', 'wcag-2.1'],
+        hasFullSupport: false,
+        id: 'legacy',
+      },
+      {
+        documentationUrl: 'http://example.com',
+        featureIds: ['discussion-page', 'lti'],
+        hasFullSupport: false,
+        id: 'piazza',
+      }];
+
+    features = [
+      { id: 'lti' },
+      { id: 'wcag-2.1' },
+      { id: 'discussion-page' },
+      { id: 'embedded-course-sections' },
+    ];
+
+    const wrapper = render(
+      <IntlProvider locale="en">
+        <FeaturesTable
+          apps={apps}
+          features={features}
+        />
+      </IntlProvider>,
+    );
+    container = wrapper.container;
+  });
+
+  test('displays a table, a thead, a tbody', () => {
+    expect(container.querySelectorAll('table')).toHaveLength(1);
+    expect(container.querySelectorAll('table > thead')).toHaveLength(1);
+    expect(container.querySelectorAll('table > tbody')).toHaveLength(1);
+  });
+
   test('displays a row for each available feature', () => {
+    expect(container.querySelectorAll('tbody > tr')).toHaveLength(features.length);
 
+    features.forEach((feature) => {
+      const featureNodes = queryAllByText(
+        container, messages[`featureName-${feature.id}`].defaultMessage,
+      );
+      expect(featureNodes.map(node => node.closest('tr'))).toHaveLength(1);
```

Comment:
What are you trying to test here?

Reply:
Could you follow up on @asadazam93 comment here! Is there a better way to test this other than checking the length of the row!?